qiime2/core/transform.py

```diff
@@ -171,10 +171,10 @@ def _wrap_transformer(self, in_, out_, wrap_input=False,
             return None
 
         if wrap_input:
-            transformer = self._wrap_input(transformer)
+            transformer = in_._wrap_input(transformer)
 
         if wrap_output:
-            transformer = self._wrap_output(transformer)
+            transformer = out_._wrap_output(transformer)
```

Comment:
This is the only *real* change. `in_` and `self` are the same object technically, I just renamed for symmetry.